### PR TITLE
move kubernetes related settings

### PR DESCRIPTION
**Pull request by nimrodshn**
Issue #2 
Moving kubernetes related settings
cc: @cben @simon3z

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -46,3 +49,18 @@
     :user:
 :container_scanning:
   :scanning_job_timeout: 20.minutes
```
Comment (moolitayer) [inline config/settings.yml]: We are missing here:
```
  :concurrent_per_ems: 3
```

**Comment by moolitayer** [comment]
Do we have a PR for manageiq I can test this with?

**Comment by nimrodshn** [comment]
@moolitayer nice!

**Comment by nimrodshn** [comment]
@moolitayer PTAL

**Comment by moolitayer** [comment]
Thanks @enoodle 
:tada:

**Pull request merged**